src/constants/cookieDetails.js

```diff
@@ -12,5 +12,5 @@ governing permissions and limitations under the License.
 
 export default {
   ALLOY_COOKIE_NAME: "adobe_alloy", // TODO: Rename this cookie
-  ALLOY_COOKIE_TTL: 180
+  ALLOY_COOKIE_EXPIRES: 180
```

Comment:
Can you please change the name to specify the type? is `180` in days?
I already have a ticket for it, so 2 birds, 1 PR: CORE-32680

Reply:
Yeah, this ran across my mind when I was changing it and now I regret not doing it. I renamed it to match the `expires` option of `cookie-js`, but I agree it should have the unit.